### DISCUSSION TITLE
Create a new keyboard popup

### DISCUSSION THREAD

**Discussion by ichnilatis-gr** [koreader/koreader#8310] [Q&A] [answered]
Hi,
I would like to make a new popup for the key  "´" in line 88, column 4 below: 
https://github.com/koreader/koreader/blob/d8d0066ac0f3c2ff2e3b215f6fd9f2b5f1a64db0/frontend/ui/data/keyboardlayouts/el_keyboard.lua#L82-L92

but I'm not sure about the procedure; I'm afraid of making a mess. Could anybody help me?

Thanks!

**Comment by Frenzie** [comment, marked as answer]
Replace that `"´"` with a variable `acu` (for acute accent) or something. The main reason I padded some variables with underscores in the original design was so they'd have the same width as just the character.

So for example, I made `"A"` into `_A_`.

Not a requirement, but I thought it looked more balanced that way. ;-)

That `acu` should be define above, presumably on line 60.

`local acu = el_popup.acu`

And in turn, in the el_popup file you merely need to add:

```lua
    acu = {
        "´", -- the main character
        northwest = "fancy new stuff",
    },
```

**Reply by ichnilatis-gr** [reply]
Thank you! I'll try it.
